types/FormatHelpers.d.ts

```diff
@@ -73,5 +73,5 @@ export interface FormatHelpers {
     options: object,
     objectType: string,
     transformGroup: string,
-  ) => string;
+  ) => object;
```

Comment:
```suggestion
  ) =>  {
    objectType?: string;
    import?: string[];
    accessControl?: string;
};
```
To be even more specific